.changeset/whole-mirrors-throw.md

```diff
@@ -0,0 +1,5 @@
+---
+"@liam-hq/schema": patch
+---
+
+🐛 fix(schema): support chained methods like pgTable().enableRLS().$comment()
```

Comment:
I want to make the Change Log easier to understand. Is it okay to change it as follows?

```suggestion
🐛 fix(Drizzle parser): support chained methods like pgTable().enableRLS().$comment()
```